rh-tictactoe/src/app/screens/Stats/index.js

```diff
@@ -1,5 +1,33 @@
 import React from 'react';
 
-const Stats = () => <h1>Stats</h1>;
+import Stats from './layout.js';
 
-export default Stats;
+class StatsContainer extends React.Component {
+  matches = JSON.parse(localStorage.getItem('matches'));
+  xCounter = 0;
+  oCounter = 0;
+  renderMatches = (match, matchId) => {
+    if (match.winner === 'X') this.xCounter += 1;
+    if (match.winner === 'O') this.oCounter += 1;
+    return (
+      <tr key={matchId}>
+        <td>
+          Match {matchId + 1}: {match.date}
+        </td>
+        <td>{match.winner}</td>
+      </tr>
+    );
+  };
+
+  render() {
+    return (
+      <Stats
```

Comment:
Just because this component has only renders, don't create a layout file.
Set the style in here